test/org/jetbrains/plugins/ideavim/RegisterActionsTest.kt

```diff
@@ -79,7 +79,7 @@ class RegisterActionsTest : VimTestCase() {
     val after = "I f${c}ound it in a legendary land"
     var motionRightAction: ActionBeanClass? = null
     doTest(keys, before, after, CommandState.Mode.COMMAND, CommandState.SubMode.NONE) {
-      motionRightAction = VIM_ACTIONS_EP.extensions().findAny().get()
+      motionRightAction = VIM_ACTIONS_EP.extensions().filter { it.actionId == "VimPreviousTabAction" }.findFirst().get()
```

Comment:
The PR is good, but what is this change about?

Reply:
It makes the action we're using explicit, rather than relying on the order of the actions in `VimActions.xml`. The `findAny` function is documented as returning "any" item from the collection, but in practice it was returning the first item, which just happened to be the "previous tab action". I moved that action from the `motion.tabs` package to `windows.tab`, and moved the registration in `VimActions.xml` to match, and that broke the assumption on the order of the actions.